MetriCam2/MetriCam2.NetStandard.csproj

```diff
@@ -23,12 +32,11 @@
 
   <ItemGroup>
     <Folder Include="Properties\" />
-    <Folder Include="Resources\" />
   </ItemGroup>
 
   <ItemGroup>
     <Reference Include="Metrilus.Util.NetStandard">
-      <HintPath>..\..\Util\bin\ReleaseUnix\netstandard2.0\Metrilus.Util.NetStandard.dll</HintPath>
+      <HintPath>..\libraries\Metrilus.Util.NetStandard.dll</HintPath>
```

Comment:
`..\libraries\Metrilus.Util.NetStandard.dll` doesn't exist in the project tree. Should it be added?